### PR TITLE
generate code room

### PR DESCRIPTION
**Pull request by aekix**
| Q             | A
| ------------- | ---
| Is bugfix?    | No
| New feature?  | Yes
| Fixed issues  | Fixes # if any

Optional description...

### CODE REVIEW

File: backend/controllers/GameController.js [hunk 1]
```diff
@@ -2,6 +2,21 @@ const MainController = require('./mainController/MainController');
...
 
+    generateCodeRoom = () => {
+        let codeRoom = "";
+        const possible = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789";
+        //tableau à remplir des codes room déja réservé par d'autre rooms
+        let arrayCodeRoom = [];
+
+        while (arrayCodeRoom.includes(codeRoom) || codeRoom === "") {
+            codeRoom = "";
+            for (var i = 0; i < 6; i++) {
```
Comment (alexisljn) [inline backend/controllers/GameController.js:13]: You should replace var by let.

**Review by alexisljn — CHANGES_REQUESTED** [review]

File: backend/controllers/GameController.js [hunk 2]
```diff
@@ -2,6 +2,21 @@ const MainController = require('./mainController/MainController');
 
 class GameController extends MainController {
 
+    generateCodeRoom = () => {
+        let codeRoom = "";
+        const possible = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789";
```
Comment (EmileCalixte) [inline backend/controllers/GameController.js:7]: Code room will be case-insensitive, so you can remove capital letters

**Review by EmileCalixte — CHANGES_REQUESTED** [review]

File: backend/controllers/GameController.js (hunk 1 above)
Comment (aekix) [reply backend/controllers/GameController.js]: Done

File: backend/controllers/GameController.js (hunk 2 above)
Comment (aekix) [reply backend/controllers/GameController.js]: Done

**Review by EmileCalixte — APPROVED** [review]
Great !

**Review by alexisljn — APPROVED** [review]
Nice work

**Pull request merged**